spec/requests/internal_api/v1/clients_controller_spec.rb

```diff
@@ -39,6 +41,15 @@
       end
     end
 
+    context "when user is an book keeper" do
```

Comment:
```suggestion
    context "when user is a book keeper" do
```
Check for grammar in all spec descriptions. I see the B of book keeper is unnecessarily capitalized at some places